.github/workflows/autopr_merged.yaml

```diff
@@ -0,0 +1,90 @@
+name: "Tag and release to production"
+
+on: 
+  pull_request:
+    types: [closed]
+
+jobs:
+  tag-release:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.base.ref == 'main' && github.event.pull_request.merged && (contains(github.event.pull_request.title, '[AUTO-PR]') || contains(github.event.pull_request.title, '[MANIFEST]')) }}
+    
+    steps:
+      - name: Obtain a Notify PR Bot GitHub App Installation Access Token
+        run: |
+          TOKEN="$(npx obtain-github-app-installation-access-token@1.1.0 ci ${{ secrets.GH_APP_CREDENTIALS_TOKEN }})"
+          echo "::add-mask::$TOKEN"
+          echo "GITHUB_TOKEN=$TOKEN" >> $GITHUB_ENV
+
+      - name: Bump version and push tag
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v6.0
+        with:
+          github_token: ${{ env.GITHUB_TOKEN }}
+          release_branches: main
+
+      - name: Checkout main branch
+        uses: actions/checkout@v2
+
+      - name: Get current Prod Version
+        run: |
+          CURRENT_VERSION=`cat VERSION`
+          echo "CURRENT_VERSION=$CURRENT_VERSION" >> $GITHUB_ENV
+
+      - name: Branch protection OFF
+        uses: octokit/request-action@v2.x
+        with:
+          route: PUT /repos/:repository/branches/main/protection
+          repository: ${{ github.repository }}
+          required_status_checks: |
+            null
+          enforce_admins: |
+            null
```

Comment:
Another question - if we just disabled `enforce_admins` and the bot performing this action had an `admin` token, it would lower the risk of branch protection being disabled since it would only impact `admins` if it didn't get re-enabled properly (or there was a race-condition).

On the other-hand, it would mean our bot was `admin` scoped.  What do you think?

Reply:
let me try it in my test repo!

Reply:
thinking about this again, i don't think giving the bot admin is a good idea just in-case the token somehow gets leaked. This would allow adding users, changing secrets, etc.

Reply:
makes sense - I couldn't decide which was worse 😅